src/gift-outline.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24"><defs><path d="M5 22a2 2 0 01-1.995-1.85L3 20v-6H2v-4a2 2 0 011.85-1.995L4 8l2.92.002a6.645 6.645 0 01-1.064-1.555c-.599-1.197-.824-2.374-.853-3.25L5 3V2h1c2.89 0 4.87 1.239 6 3.28C13.13 3.238 15.11 2 18 2h1v1c0 .891-.211 2.158-.856 3.447a6.645 6.645 0 01-1.064 1.555L20 8a2 2 0 011.995 1.85L22 10v4h-1v6a2 2 0 01-1.85 1.995L19 22H5zm6-8H5v6h6v-6zm8 0h-6v6h6v-6zm-8-4H4v2h7v-2zm9 0h-7v2h7v-2zm-3.14-5.91c-1.568.262-2.584 1.093-3.216 2.357-.185.37-.327.763-.432 1.163l-.072.3c1.568-.262 2.584-1.093 3.216-2.357.23-.463.396-.96.504-1.464zm-9.72 0c.108.503.273 1 .504 1.463.597 1.193 1.537 2.002 2.96 2.309l.256.049-.072-.301a6.005 6.005 0 00-.432-1.163C9.724 5.183 8.708 4.352 7.14 4.09z" id="a"/></defs><use fill="#000" fill-rule="nonzero" xlink:href="#a"/></svg>
```

Comment:
- Please remove unnecessary `<defs>` `</defs>`
- Remove the `id="a"`
- Remove `<use fill="#000" fill-rule="nonzero" xlink:href="#a"/>`